addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java

```diff
@@ -258,6 +340,14 @@ private synchronized MaxCubeBridgeHandler getMaxCubeBridgeHandler() {
         return this.bridgeHandler;
     }
 
+    private Device getDevice() throws NullPointerException {
+        Device device = getMaxCubeBridgeHandler().getDevice(maxDeviceSerial);
+        if (device == null) {
+            throw new NullPointerException();
```

Comment:
this is a rather unusual construct. Why not return null and handle that in the caller?